cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -3127,142 +3129,457 @@ func TestMutateGlobalPullSecret(t *testing.T) {
 			expectedErr: fmt.Errorf("failed to parse the original secret: failed to get content from nil secret"),
 		},
 		{
-			name: "basic case: expired auth is replaced",
+			name: "not mutated if the auth is still valid",
+			original: &coreapi.Secret{
```

Comment:
Could we switch the order back?
If you want you may change it with another PR.

Reply:
I ordered them as they were before, new tests are at the end

Reply:
The previous order was secret and then original and now it is the other way around.